src/lib/createPollBlocks.ts

```diff
@@ -31,6 +31,15 @@ export function createPollBlocks(block: BlockBuilder, question: string, options:
 
     block.addDividerBlock();
 
+    // Array of vote quantities
+    const voteQuantities = poll.votes.map(vote => vote.quantity)
+    
+    // Forms array of option indices with maximum votes (more than 1 option can be max-voted)
+    const maxVoteIndices = voteQuantities.reduce(function(ind: number[], el, i) { 
```

Comment:
Will recommend to use arrow function instead - makes the code look much cleaner

Reply:
Resolved. Thanks